Common/Tests/Utilities/TestExtensions.cs

```diff
@@ -24,6 +24,12 @@
 
 namespace TestUtilities {
     public static class TestExtensions {
+        public const int CORE_UNIT_TEST = 0;          // Run on every PR request going into master
+        public const int IMPORTANT_UNIT_TEST = 1;     // Run on every commit into master
```

Comment:
I'm a bit hesitant to use "Priority" because these numbers don't inherently mean anything. When the UI tests are implemented, they can't have the same priority as unit tests which implies they are less or more priority than unit test which is not really the case. I've changed the 2nd variable name to "SUPPLEMENTARY_UNIT_TEST". The new name and comment should be sufficiently descriptive.

Reply:
When it comes to priority, P0, P1, P2 is the standard way of referring to them. If you look at the issues labels on our repo, we have priority:P0, priority:P1, priority:P2 and priority:release-blocking.

priority:release-blocking was an attempt at giving a name to a priority, thinking that it would be more understandable. It wasn't. No one is used to that, and now it's been abandoned, partly due to the inconsistency with other repos our team owns which use P0 as the highest possible priority.

Core is a term used in the product to represent the core part of the extension, there are folders named core, so someone could think that it somehow refers to that.

If you want different values for unit test and ui tests, you can still achieve that while having priorities that are easily ranked.

UI tests can use a different constant class:

```
public static class UnitTestPriority {
    public const int P0 = 0;
    public const int P1 = 1;
    public const int P2 = 2;
    public const int P0_FAILING = 11;
    public const int P1_FAILING = 21;
    public const int P2_FAILING = 31;
}

public static class UITestPriority {
    public const int P0 = 0; // or whatever integer value you want
    public const int P1 = 1;
    public const int P2 = 2;
    public const int P0_FAILING = 11;
    public const int P1_FAILING = 21;
    public const int P2_FAILING = 31;
}
```

Or they could be different constants in the same class:

```
public static class TestPriority {
    public const int UNIT_TEST_P0 = 0;
    public const int UNIT_TEST_P1 = 1;
    public const int UNIT_TEST_P2 = 2;
    public const int UNIT_TEST_P0_FAILING = 11;
    public const int UNIT_TEST_P1_FAILING = 21;
    public const int UNIT_TEST_P2_FAILING = 31;

    public const int UI_TEST_P0 = 0; // or whatever integer value you want
    public const int UI_TEST_P1 = 1;
    public const int UI_TEST_P2 = 2;
    public const int UI_TEST_P0_FAILING = 11;
    public const int UI_TEST_P1_FAILING = 21;
    public const int UI_TEST_P2_FAILING = 31;
}

```

Reply:
Changed.